dbclient/HiveClient.py

```diff
@@ -58,6 +58,27 @@ def launch_cluster(self):
                 raise Exception("Could not launch cluster. Verify that the --azure flag or cluster config is correct.")
             self.wait_for_cluster(c_info['cluster_id'])
             return c_info['cluster_id']
+    
+    def edit_cluster(self, cid, iam_role):
+        """Edits the existing metastore cluster
+        Returns cluster_id"""
+        version = self.get_latest_spark_version()
+        json_payload = {
```

Comment:
This json payload should load from the `data/aws_config.json` files that contain the cluster configurations, then add the IAM role to the AWS attributes. This will reduce the amount of hard coded attributes within the code.